appcenter-data/android/src/main/java/com/microsoft/appcenter/reactnative/data/AppCenterReactNativeDataModule.java

```diff
@@ -28,6 +28,20 @@
 
 public class AppCenterReactNativeDataModule extends BaseJavaModule {
 
+    private static final String DESERIALIZED_VALUE_FIELD = "deserializedValue";
+
+    private static final String JSON_VALUE_FIELD = "jsonValue";
```

Comment:
I'm not such a fan of putting `_FIELD` on field names, it's redundant.

Same applies to the other constants here

Reply:
Agree with @jwallra, remove `_FIELD`.

Reply:
I would use `KEY` instead of `FIELD` if we're going to keep it

Reply:
[AppCenterReactNativeCrashesModule.java](https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-crashes/android/src/main/java/com/microsoft/appcenter/reactnative/crashes/AppCenterReactNativeCrashesModule.java) uses `FIELD`. I agree with @jwallra, `KEY` seems to be a more appropriate suffix.

Reply:
Changed to `KEY` instead.